universum/modules/artifact_collector.py

```diff
@@ -112,7 +112,10 @@ def create_text_file(self, name):
                     raise CriticalCiException(text)
 
             self.file_list.add(file_name)
-            file_path = self.automation_server.artifact_path(self.artifact_dir, os.path.basename(file_name))
+            try:
+                file_path = self.automation_server.artifact_path(self.artifact_dir, os.path.basename(file_name))
```

Comment:
It seems there are two issues here:
1. Printing any path doesn't make much sense for GitHub Actions at the moment
2. The logic is based on exceptions
We cannot do much about item 1. But we can just return the name from the GithubServer class instead of raising an exception. The end result will be the same (or almost the same if you cannot recover the original path inside GithubServer).